server/commands/rollback.go

```diff
@@ -107,9 +107,6 @@ func liteBlockManager(context context.Context, cfg *config.Config, dymintConf *d
 		return nil, err
```

Comment:
context here is unused:

https://github.com/dymensionxyz/dymension-rdk/pull/495/files#diff-de55ef80bcd4644a0634b2088d2852e8a18737f56eab737b216e22b8a57d4c18L88